labgraph/lsl_node/lsl_poller_node.py

```diff
@@ -0,0 +1,37 @@
+from pylsl import StreamInlet, resolve_stream
+from ..graphs import AsyncPublisher, Config, Node, Topic, publisher
+from ..util.logger import get_logger
+from ..lsl_node import LSLMessage
+import asyncio
+
+
+logger = get_logger(__name__)
+
+
+class LSLPollerConfig(Config):
+    type: str = 'labgraph'
```

Comment:
Are the following attributes applicable to this config: number_channels, sample_rate

Reply:
No, they aren't the poller just needs the type of the stream to connect to